portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro

```diff
@@ -236,6 +236,25 @@ definition {
 		return "${json}";
 	}
 
+	macro _updateEmployee {
```

Comment:
tom improve the readability of the method you might want to implement:
`Variables.assertDefined(parameterList = "${parameter1},${parameter2}");`
Being the parameters a list of all the variables that are necessary to perform the operation described in the method.

Reply:
how about updateEmployeeById?